.husky/pre-commit

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+# . "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
+
+for file in *; do
+  if [[ -d "$file" ]] # is a directory?
+  then
+    dir=$file
+    if ! [[ " node_modules ecommerce-one-click " =~ " $dir " ]]
```

Comment:
I'd at least just prompt and say something like:
```
Running pre-commit hook, this will overwrite any config files created in samples, proceed? [Y/n]
```